app/code/Magento/Developer/Console/Command/QueryLogEnableCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Magento\Framework\DB\Logger\LoggerProxy;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\DeploymentConfig\Writer;
+use Magento\Framework\Config\File\ConfigFilePool;
+
+class QueryLogEnableCommand extends Command
+{
+    /**
+     * input parameter log-all-queries
+     */
+    const INPUT_ARG_LOG_ALL_QUERIES = 'log-all-queries';
+
+    /**
+     * input parameter log-query-time
+     */
+    const INPUT_ARG_LOG_QUERY_TIME = 'log-query-time';
```

Comment:
please remove "log-" prefix on all these options names
rename option to query-time-threshold